CHANGELOG.md

```diff
@@ -120,6 +120,7 @@ This release contains the [PDV refactor (#4111)](https://github.com/hasura/graph
 - server: accept only non-negative integers for batch size and refetch interval (close #5653) (#5759)
 - server: fix bug which arised when renaming a table which had a manual relationship defined (close #4158)
 - server: limit the length of event trigger names (close #5786)
+- server: Configurable websocket keep-alive interval (fix #3539) 
```

Comment:
Would be good to add the flag and env variable here.

Reply:
hey @tirumaraiselvan changelog.md is updated :+1: